cmd/auth/main.go

```diff
@@ -69,6 +70,14 @@ func main() {
 		log.Fatalf("failed to init logger: %s", err)
 	}
 
+	instanceID := cfg.InstanceID
+	if instanceID == "" {
+		instanceID, err = uuid.New().ID()
+		if err != nil {
+			log.Fatalf("Failed to generate instanceID: %s", err)
+		}
+	}
```

Comment:
@ianmuchyri

This is repeated in all `main.go` files, 
Why don't we move to `internal/clients/jaeger/tracer.go` ?

Reply:
@arvindh123 Instance ID is not only used in Jeager. It's also used in health check endpoint and is, generally, related to higher-level config. We should probably create the highest-level config that holds the instance id, service name, and tracing URl. Maybe some others. We'll investigate this in a separate PR.

Reply:
Yes,  I agree